### PR TITLE
Add create/remove gRPC api for vm

### PR DESCRIPTION
**Pull request by feiskyer**
Part of #278

### CODE REVIEW

File: serverrpc/vm.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"github.com/golang/glog"
+	"github.com/hyperhq/hyperd/types"
+	"golang.org/x/net/context"
+)
+
+// VMCreate implements POST /vm/create
+func (s *ServerRPC) VMCreate(ctx context.Context, req *types.VMCreateRequest) (*types.VMCreateResponse, error) {
+	glog.V(3).Infof("VMCreate with request %s", req.String())
+
+	if req == nil || req.Cpu == 0 || req.Memory == 0 {
```
Comment (laijs) [inline serverrpc/vm.go]: why "req == nil" is needed?

Comment (feiskyer) [inline serverrpc/vm.go]: If req is indeed nil, then `req.Cpu == 0 || req.Memory == 0` will panic.

Comment (laijs) [inline serverrpc/vm.go]: how to make it happen

Comment (feiskyer) [inline serverrpc/vm.go]: The client may pass in a nil req.

**Pull request merged**